geometry_msgs/PoseStamped pose
string class_str
int8 id